tox.ini

```diff
@@ -247,9 +260,10 @@ max-line-length = 88
 
 [travis]
 python =
-  2.6: py26,extra
-  2.7: py27,flake8,pylint,extra
-  3.5: coveralls,molecule,extra
-  3.6: py36,black,extra
-  3.7: py37,extra
-  3.8: py38,extra
+  2.6: py26,custom
+  2.7: py27,flake8,pylint,custom
+  3.5: coveralls,custom
+  3.6: py36,black,coveralls,custom
+  3.7: py37,coveralls,custom
+  3.7-dev: py37,coveralls,custom
+  3.8-dev: py38,coveralls,custom
```

Comment:
Is it correct to remove python 3.8 here?

Reply:
I am not sure. Python 3.8.0 is the latest release of Python now and will be shortly supported in [Fedora](https://fedoraproject.org/wiki/Changes/Python3.8). Also it seems that it is [supported](https://docs.travis-ci.com/user/languages/python/) by Travis, so I propose to add `3.8` line besides `3.8-dev` line.

Reply:
do we need both 3.7-dev and 3.8-dev ? also, do we need coveralls in multiple places?

Reply:
merged the python version change in #137